org.civicrm.bootstrapcivihr/scss/mixins/_spinner.scss

```diff
@@ -0,0 +1,39 @@
+// Spinner related variables
+$default-height: 32px;
+$default-width: 32px;
```

Comment:
why would the width be ever different than the height? the spinner should always be a square